src/Storages/LiveView/StorageLiveView.cpp

```diff
@@ -356,6 +410,7 @@ bool StorageLiveView::getNewBlocks()
             (*blocks_ptr) = new_blocks;
             (*blocks_metadata_ptr) = new_blocks_metadata;
             updated = true;
+            writeNewBlocksToTargetTable(context);
```

Comment:
After this change, method `getNewBlocks` looks too complex. Now it writes blocks somewhere, which is not obvious from it's name. It's also not obvious that we need to write them every time. 
I think it's reasonable to move this call outside.

Reply:
Ok, I will move it to where `getNewBlocks` is called.